packages/atomicHelpers/persistentStorage.ts

```diff
@@ -0,0 +1,97 @@
+import type { PersistentAsyncStorage } from "@dcl/kernel-interface"
+
+declare var window: any
+
+class PersistentLocalStorage implements PersistentAsyncStorage {
+  storage: any
+  constructor(storage: any){
```

Comment:
Why does this constructo receives a "storage"? Should it have types?

Reply:
That implementation of LocalPersistentStorage is a wrapper of localStorage. Maybe it should be in another file.
localStorage is a Storage type from DOM library.

LocalPersistentStorage keeps compatibility with kernel usages without storage option.